### PR TITLE
Update to Popcorn SDK 6

### PR DESCRIPTION
**Pull request by mediaminister**

### CODE REVIEW

File: resources/lib/vtmgo/vtmgostream.py [hunk 1]
```diff
@@ -67,9 +68,19 @@ def get_stream(self, stream_type, stream_id):
...
+            license_provider = stream_info.get('drm', {}).get('com.widevine.alpha', {}).get('provider')
+            if license_provider == 'drmtoday':
+                license_key = self.create_license_key(
+                    license_url,
+                    key_headers={
+                        'x-dt-auth-token': stream_info.get('drm', {}).get('com.widevine.alpha', {}).get('drmtoday', {}).get('authToken'),
+                        'Content-Type': 'application/octet-stream',
+                    },
+                    response_value='JBlicense'
+                )
```
Comment (michaelarnauts) [inline resources/lib/vtmgo/vtmgostream.py:83]: I think you need to do an else here for anvato and set the license_key car instead of license_url above.

Comment (mediaminister) [reply resources/lib/vtmgo/vtmgostream.py:88]: You're right, there were more changes needed to fully upgrade to sdk version 6.

Comment (michaelarnauts) [reply resources/lib/vtmgo/vtmgostream.py:85]: Is it the same as the changes for Streamz?

Comment (mediaminister) [reply resources/lib/vtmgo/vtmgostream.py:85]: Mostly, but livestreams are still using the old anvato format.

**Comment by michaelarnauts** [comment]
Awesome! Thanks!

**Pull request merged**